ateam_ai/src/ateam_ai_node.cpp

```diff
@@ -194,6 +196,11 @@ class ATeamAINode : public rclcpp::Node
       }
     }
 
+    // Get current game state for world
+    // 1. Get command
+    world_.referee_info.running_command = game_state_listener_.GetGameCommand();
+    // 2. Get the game stage
+    world_.referee_info.current_game_stage = game_state_listener_.GetGameStage();
```

Comment:
Style nit: These commands don't add any information for the reader. The function name already tells me it's getting the game command, for example.

Reply:
Kept the brief "Get current game state for world", but removed the subsections, since the other code in this node has similar comments.